### PR TITLE
Planqk colors

### PR DESCRIPTION
**Pull request by mhinkie**
Changes color scheme to PlanQK colors.

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -15,11 +15,15 @@
...
-    <mat-icon>library_books</mat-icon>
+  <mat-toolbar color="primary" id="pa_header">
+    <a href="https://platform.planqk.de/my-content">
+    <img src="https://platform.planqk.de/img/PlanQK_Logo_Weiss_Beta_Version.5b689144.svg" alt="PlanQK Logo" width="250"
+         style="max-width: 7rem; margin-top: 1rem">
+    </a>
     <span style="margin-left: 1rem">Pattern Atlas</span>
     <span class="spacer"></span>
     {{welcomeText}}
+    <a mat-stroked-button href="https://platform.planqk.de/my-content">Back to the PlanQK platform</a>
```
Comment (lharzenetter) [inline src/app/app.component.html:26]: I guess we should make this configurable. In the local or other environment, we should not show this. Thus, maybe add a corresponding flag which only shows this if the the flag is set.

**Review by lharzenetter — CHANGES_REQUESTED** [review]
Please add the configuration for the back to the platform button

File: src/app/app.component.ts [hunk 2]
```diff
@@ -65,7 +68,10 @@ export class AppComponent implements OnInit {
 
   ngOnInit(): void {
     this.configService.getConfigurationFromBackend().subscribe(
-      () => (this.loading = false),
+      () => {
+        this.loading = false;
+        this.editingFromConfigServer = this.configService.configuration.features[UiFeatures.EDITING];
```
Comment (lharzenetter) [inline src/app/app.component.ts:73]: I guess we should add a separate feature that indicates if the PlanQK Logo should be displayed or not.

**Review by lharzenetter — CHANGES_REQUESTED** [review]

File: src/app/app.component.ts (hunk 2 above)
Comment (mhinkie) [reply src/app/app.component.ts]: Added a new configuration entry that defaults to true (if the config server is not available) and indicates if planqk-ui should be used

**Review by lharzenetter — APPROVED** [review]

**Review by lharzenetter — APPROVED** [review]

**Pull request merged**